exampleSite/content/nesting/content.md

```diff
@@ -0,0 +1,39 @@
++++
+fragment = "content-single"
+#disabled = true
+date = "2017-10-05"
+weight = 100
+#background = ""
+description = "Nested Page"
```

Comment:
Probably should be `title`?

Reply:
It's the mirror of https://github.com/okkur/syna/pull/118/files#diff-fadb7b097ab29fdd0268523ca6c0e097L5

Reply:
Should probably add title though.

Reply:
Yeah, but I think that was a leftover mistake I would assume. We don't use description at all so it probably is save to migrate that to title.